x-pack/plugins/apm/common/agent_configuration/all_option.ts

```diff
@@ -6,13 +6,17 @@
  */
 
 import { i18n } from '@kbn/i18n';
-
+import { ENVIRONMENT_ALL } from '../environment_filter_values';
 export const ALL_OPTION_VALUE = 'ALL_OPTION_VALUE';
 
 // human-readable label for service and environment. The "All" option should be translated.
 // Everything else should be returned verbatim
 export function getOptionLabel(value: string | undefined) {
-  if (value === undefined || value === ALL_OPTION_VALUE) {
+  if (
+    value === undefined ||
+    value === ALL_OPTION_VALUE ||
+    value === ENVIRONMENT_ALL.value
+  ) {
```

Comment:
Hmm... This seems like we have started to mix `getOptionLabel` and `getEnvironmentLabel`. Why are both `ALL_OPTION_VALUE` and `ENVIRONMENT_ALL.value` possible values? Shouldn't it only be one of them?

Reply:
I've realised this introduces another bug, I'll fix it

Reply:
updated here: https://github.com/elastic/kibana/pull/133616/commits/cf04974141e1241ec1c15a7ad627005007b1b6f6